pages/create/index.page.js

```diff
@@ -1,35 +1,78 @@
-import Button from "../../components/button/button";
-import Input from "../../components/input/input";
-import Select from "../../components/select/select";
-import { Container, Form, Title } from "./create.styles";
 import { useForm } from "react-hook-form";
 import { number, object, string, date, ref } from "yup";
 import { yupResolver } from "@hookform/resolvers/yup";
+import { useContractWrite, usePrepareContractWrite } from "wagmi";
+import useDebounce from "../../hooks/useDebounce";
+
+import Select from "../../components/select/select";
+import crowdfundingConfig from "../../crowdfunding.config.json";
+
+import { Container, Form, Title } from "./create.styles";
+
+const validationSchema = object().shape({
+  title: string().required(),
+  subtitle: string().required(),
+  story: string().required(),
+  token: string().required(),
+  fundGoal: number().required(),
+  startDate: date()
+    .required()
+    .min(new Date(), "Date should be greater than now"),
+  endDate: date()
+    .required()
+    .min(ref("startDate"), "End date should be after start date"),
```

Comment:
Don't know about `yup` but seems that `endDate` accepts a value equals than `startDate`, could it be? 


I suppose that it should be `min(ref("startDate") + 1)` or something, to force the >= condition